### ISSUE TITLE
Create the "Top performing keywords" key metric widget tile

### ISSUE THREAD

**Issue by jimmymadon** [google/site-kit-wp#6251]
## Feature Description

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation Brief

* 

### Test Coverage

* 

## QA Brief

* 

## Changelog entry

*

**Comment by tofumatt** [comment]
> Fetch 100 of the top search queries from from the [Google Search Console API](https://developers.google.com/webmaster-tools/v1/searchanalytics/query) which are automatically sorted by Clicks. Then select the top 3 with the highest CTR.

Makes sense, but the ACs should be a bit more clear about the method, just for clarity's sake.

I assume we'll do this via the WP REST API/Site Kit API—do we need to add a new endpoint? If we can just use an existing one it's fine to say that, but if we need to add one we should be clear in the ACs, I think 😅

**Comment by tofumatt** [comment]
I think here it's helpful to mention it in the ACs, thanks! 🙂 

ACs look good, moving to IB 👍🏻

**Comment by tofumatt** [comment]
IB ✅

**Comment by mohitwp** [comment]
### QA Update ✅ 

- Tested on dev environment.
- Verified "Top performing keywords" key metric widget tile with Figma design.
- Verified "Top performing keywords" key metric widget tile when search console is in zero or gathering state.
- Verified Story book story for 'Popular Keywords Widget'.
- Verified loading state as per storybook.

As mentioned in QAB-
Note 1: The mobile version currently does not display Key Metric tiles in a single column. A new issue is or will be created for this.
Note 3: Designing the error state for widget tiles is out of the scope of this issue. A new issue is or will be created for this.


<details>

![image](https://github.com/google/site-kit-wp/assets/94359491/fa10fd49-0262-43bb-8a4c-fac441320b74)

![image](https://github.com/google/site-kit-wp/assets/94359491/d23b8260-35fe-4a8f-a913-7c514e9b0b65)

![image](https://github.com/google/site-kit-wp/assets/94359491/c150dacf-ab31-46be-83f5-4d799802e6c5)

![image](https://github.com/google/site-kit-wp/assets/94359491/7595d5fb-ab1c-4139-987b-7afa59e0d4a3)

![image](https://github.com/google/site-kit-wp/assets/94359491/85d26dc8-6f45-4e5b-9537-9519f4dc2d67)

</details>
